### PR TITLE
Added native About window

### LINKED ISSUE #16: About window could use more info
The "About" window currently is just a splash screen of CapitalBe.
If there was more info added, that'll be great!

### PR DESCRIPTION
**Pull request by alwayslivid**
Fixes #16 and also uses a native BAboutWindow, per @humdingerb's wishes, as stated in the issue.

### CODE REVIEW

**Comment by scottmc** [comment]
This was written by DarkWyrm, I think most of the others have just maintained it since then.  I don't think they added any new features, etc.  Also doublecheck the commits history to see if those you listed wrote the fixes or just approved someone else's pull request.  I don't recall writing any code for this.

**Comment by alwayslivid** [comment]
> This was written by DarkWyrm, I think most of the others have just maintained it since then. I don't think they added any new features, etc. Also doublecheck the commits history to see if those you listed wrote the fixes or just approved someone else's pull request. I don't recall writing any code for this.

I mistakenly added your name because your name seems to only be there only due to PR request approvals. I will promptly remove it, although other than that, I added DarkWyrm as the copyright holder. This is consistent with other occasions where the native About window is used in some other Haiku software.

**Comment by owenca** [comment]
Your fork is 1 commit behind now. Please rebase it and then squash your commits.

**Comment by alwayslivid** [comment]
Is there anything else that I need to address?

**Comment by korli** [comment]
Looks good to me. Asking humdinger whether the issue is solved for him.

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -239,6 +239,27 @@ MainWindow::~MainWindow(void)
...
+		"Thomas Schmidt",
+		"waddlesplash",
+		NULL
+	};
+
+	abwin->AddCopyright(2009, "DarkWyrm");
+	abwin->AddAuthors(authors);
+	abwin->AddText("Distributed under the terms of the MIT License");
+	abwin->Show();
+
```
Comment (humdingerb) [inline src/MainWindow.cpp]: Remove empty line.

**Pull request merged**